test/spec/dataviews/histogram-dataview-model.spec.js

```diff
@@ -340,4 +340,70 @@ describe('dataviews/histogram-dataview-model', function () {
       expect(this.model.get('own_filter')).toBeUndefined();
     });
   });
+
+  describe('._sumBinsFreq', function () {
```

Comment:
I'm not a bit fan of testing private methods (unless there's a very good reason to do it, and maybe there is)... In this case, why not just test this by setting different data on the `filter` and asserting that `totalAmount` and `filteredAmount` attributes are updated correctly?